src/e2e/scala/temple/SimpleE2ETest.scala

```diff
@@ -21,15 +21,25 @@ class SimpleE2ETest extends FlatSpec with Matchers {
       ),
     )
 
-    // Only one folder should have been generated
-    Files.list(basePath).count() shouldBe 1
+    // Two folders should have been generated
+    Files.list(basePath).count() shouldBe 2
```

Comment:
Can we check what they are, rather than their count? That way it will be easier to check what the test intended when it doesn’t pass.

Reply:
`Files.list(basePath)` should be a set of x and y

Reply:
I thought that was sufficiently tested by first checking there was only two, and then checking which folders they were in the following two tests?

Reply:
But I guess the error if it doesn't succeed will be that 2 != 3, if that's what you mean?

Reply:
Although this does fail when there’s an error, checking the contents of the list should lead to more readable errors (“x missing” rather than 1 != 2)